litmus-portal/graphql-server/pkg/chaos-workflow/service.go

```
@@ -366,28 +367,28 @@ func processWorkflowManifest(workflow *model.ChaosWorkFlowRequest, weights map[s
 				var meta chaosTypes.ChaosEngine
 				err := yaml.Unmarshal([]byte(data), &meta)
 				if err != nil {
-					return errors.New("failed to unmarshal chaosengine")
+					return nil, nil, errors.New("failed to unmarshal chaosengine")
```

```suggestion
					return nil, nil, fmt.Errorf("failed to unmarshal chaosengine %v", err)
```

---

Let's return the actual error as well